### PR TITLE
Update T4c parameters in README

### PR DESCRIPTION
**Pull request by moritz9**

### CODE REVIEW

File: data_pipeline/README.md [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
+* `dp03_road_graph.py`
 
-## Data Pipeline Node Data: Vehicle Counts
-<img src="../img/data_pipeline_node_data.svg">
+Run `dp03_road_graph.py`  with the following custom arguments:
+* `--custom_filter '["highway"~"motorway|motorway_link|trunk|primary|secondary|tertiary|unclassified|residential"]'`
+* `--parse_maxspeed` (needs to be enabled for Madrid)
+* `--heatmap_filter`
 
+The output files `road_graph_nodes.parquet`, `road_graph_edges.parquet`, `road_graph.gpkg` and `road_graph.graphml` will be written to `road_graph/<city>`.
```
Comment (chenkins) [inline data_pipeline/README.md:18]: Do we want to refer to `README_DATA_SPECIFICATION.md` in both repos for the minor differences in the attributes?

File: data_pipeline/README.md [hunk 2]
```diff
@@ -30,14 +29,69 @@ We will publish the code to generate the speed classifications from the traffic
...
 
 
+### Vehicle Counter Data Pipeline
+
+Vehicle loop counter data can be processed using the pipeline used for counter validations in MeTS-10: https://github.com/iarai/MeTS-10/tree/master/analysis/val02_counters
+
+This process requires the road graph data in `road_graph/<city>`.
+
+* `counters01_prepare_data.ipynb`: Downloads and processes the counter readings. Outputs `loop_counters/<CITY>/counters_normalized_{month}.parquet`
+* `counters02_match_counters.ipynb`: Merges the loop counter locations with the road graph and outputs `loop_counters/<CITY>/counters_matched.parquet`
```
Comment (chenkins) [inline data_pipeline/README.md:39]: Should it not output a new road graph with the new nodes as well? (see CODE-1 in overleaf)

File: data_pipeline/README.md [hunk 3]
```diff
@@ -30,14 +29,69 @@ We will publish the code to generate the speed classifications from the traffic
...
+
+Outputs of `dp06_speed_classes.py` will be stored as `speed_classes_<date>.parquet` to `speed_classes/<city>`.
 
-| Item               | Description                |
-|--------------------|----------------------------|
-| Road Selection     | Select OSM primary features of type "highway" in the bounding box, introduce nodes for loop counters and simplify the road graph (remove dead-ends and unconnected components etc.). |
-| Spatial Join       | Intersect OSM roads geometries with map movie cell geometries. |
-| Combination        | Combine Traffic Map values with OSM IDs. |
-| Generate CC Labels | Generate congestion classes (green=1, yellow=2, red=3) from the current segment medium speeds, the free flow speeds computed for the segment from the traffic map movies and the OSM signalled speeds. If no or not enough dynamic speed data is available, do not classify (unclassified=0). |
+With these outputs `t4c22/prepare_training_data_cc.py` can be used to generate the training labels.
```
Comment (chenkins) [inline data_pipeline/README.md:97]: Do we want to publish super-segment sampling code as well? (see (CODE-3) in overleaf)

File: data_pipeline/README.md (hunk 2 above)
Comment (moritz9) [reply data_pipeline/README.md:39]: added TODO

File: data_pipeline/README.md (hunk 3 above)
Comment (moritz9) [reply data_pipeline/README.md:97]: TODO

**Pull request merged**